content/docs/concepts/security/index.md

```diff
@@ -367,7 +367,9 @@ namespace. Policies in mesh-scope can affect all services in the mesh. To
 prevent conflict and misuse, only one policy can be defined in mesh-scope
 storage. That policy must be named `default` and have an empty
 `targets:` section. You can find more information on our
-[target selectors section](/docs/concepts/security/#target-selectors).
+[target selectors section](/docs/concepts/security/#target-selectors). By
+default if a mesh-scoped policy is not installed, mutual TLS is considered
+disabled.
```

Comment:
I think this message is vague.  It would be helpful to specify how the mesh-scoped policy is not installed?  I assume it is one of the helm flag or other ways.

Reply:
@linsun a mesh scoped policy is covered in this document: https://istio.io/docs/tasks/security/authn-policy/#globally-enabling-istio-mutual-tls.  Its simply a `MeshPolicy` CR.  Another way to word this sentence would be

```
If a global `MeshPolicy` resource is not installed, mutual TLS is disabled by default
```

Of course, this isn't quite accurate, as with `--set global.enableControlPlaneSecurity=true` the control plane ends up with a bunch of MUTUAL_TLS configuration.  It is also possible to specify per namespace and per application TLS policies, so I agree the message is vague and unconvincing given the other documentation :)

Reply:
@linsun @sdake do you have a suggestion here for a preferred update to the docs?

Reply:
Please write in active voice, based on @sdake's text, I suggest:
You can only install a global data-plane mutual TLS policy with the \`--set global.mtls.enabled=true\` Helm installation flag. If you do not specify this flag, Istio does not enable the mutual TLS for the data plane by default.